src/VolleyManagement.Backend/VolleyManagement.Domain/Dto/UserInRoleDto.cs

```diff
@@ -25,9 +25,11 @@ public UserInRoleDto()
         /// </summary>
         public string UserName { get; set; }
 
+#pragma warning disable S3956 //Use more generic collection, LINQ convertation fails, EF bug
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)

Reply:
Add a link to the defect description

Reply:
on pragma warning restore there is a link to stackoverflow

Reply:
do you think it will be easily found?